vars/openShift.groovy

```diff
@@ -84,27 +84,37 @@ def withUINode(Map parameters = [:], Closure body) {
             containerTemplate(
                 name: 'jnlp',
                 image: slaveImage,
-                alwaysPullImage: true,
                 args: '${computer.jnlpmac} ${computer.name}',
-                workingDir: workingDir,
-                resourceRequestCpu: requestCpu,
-                resourceLimitCpu: limitCpu,
-                resourceRequestMemory: requestMemory,
-                resourceLimitMemory: limitMemory,
+                resourceRequestCpu: '100m',
+                resourceLimitCpu: '300m',
+                resourceRequestMemory: '512Mi',
+                resourceLimitMemory: '1Gi',
             ),
             containerTemplate(
                 name: 'selenium',
                 image: seleniumImage,
+                resourceRequestCpu: '500m',
+                resourceLimitCpu: '1',
+                resourceRequestMemory: '1Gi',
+                resourceLimitMemory: '4Gi',
```

Comment:
Does selenium really consume this much memory? :(

Reply:
It's not selenium but browsers :)